### ISSUE TITLE
Post Author block – add option to link to author archive

### ISSUE THREAD

**Issue by jameskoster** [WordPress/gutenberg#37748]
It would be nice to have an option to make the post author serve as a link to an archive of posts by that author.

**Comment by luminuu** [comment]
Yes, preferably linking the author name would be a useful feature.

**Comment by carolinan** [comment]
I may have misunderstood but I thought the plan was split the author block into smaller blocks, and that these would then replace the current block.
https://github.com/WordPress/gutenberg/issues/35596
The post author name block is already in Gutenberg and it has this link feature.

**Comment by jameskoster** [comment]
I suppose it depends on the implementation. If the plan is to replace the internal elements of the Post Author block with the dedicated atomic blocks (IE Post Author Name, Post Author Avatar, etc), then yes we can probably close this out.

**Comment by kathrynwp** [comment]
Another request to have a link option on the Post Author block:

https://wordpress.org/support/topic/post-author-link-3/

**Comment by kathrynwp** [comment]
@c4rl0sbr4v0 Hi - could you please confirm that this (earlier) issue is a duplicate of #41554 and should be closed? Thanks.

**Comment by jameskoster** [comment]
Seems to be complete: <img width="278" height="199" alt="Image" src="https://github.com/user-attachments/assets/6fb1dad5-2a36-4647-bb69-1fe1f3379828" />
